### ISSUE TITLE
Is epsilon in BatchNormalization layer a trainable variable?

### ISSUE THREAD

**Issue by liujxing** [fchollet/keras#6225]
I used the default value of epsilon = 0.001 for my BatchNormalization layers, but the epsilon of all such layers become 0.00001 after training. Is epsilon a trainable parameter here?

**Comment by fchollet** [comment]
No, it's not: it's a python float constant.

**Comment by liujxing** [comment]
Then why do I see a different epsilon after training?
